Microsoft.Toolkit.Uwp.UI.Animations/Behaviors/CompositionBehaviorBase.cs

```diff
@@ -86,6 +91,18 @@ public double Duration
             set { SetValue(DurationProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the Easing function
```

Comment:
use <see cref="EasingType"/> so we can get a link to the easing type docs?

Reply:
Added reference to the EasingType enum in the comments